ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -30,10 +32,10 @@ Feature: Notification creation
         When I enter 1 into 'FormattedNotificationDate_Day'
         And I enter 1 into 'FormattedNotificationDate_Month'
         And I enter 2019 into 'FormattedNotificationDate_Year'
-        And I select Ashford Hospital for 'HospitalDetails_TBServiceCode'
+        And I select Birmingham & Solihull for 'HospitalDetails_TBServiceCode'
         # Wait until javascript has populated the hospital dropdown
         And I wait
-        And I select ASHFORD HOSPITAL for 'HospitalDetails_HospitalId'
+        And I select SELLY OAK HOSPITAL for 'HospitalDetails_HospitalId'

```

Comment:
These are magic strings, but I've deferred fixing them for when we're actually developing the tests